src/static/api_documentation.yml

```diff
@@ -78,6 +93,21 @@ paths:
         '503 - Service Unavailable':
           $ref: '#/components/responses/NoCeleryWorkers'
 
+  "/models/{scenarioId}/getPollution":
```

Comment:
```suggestion
  "/models/pollution/{scenarioId}":
```

Reply:
This follows the patterns from the rest of the document closer (but the patterns aren't very strong)

Reply:
This would also mean `"/models/{scenarioId}"` becomes `"/models/flood/{scenarioId}"` including in the python code